apps/root/src/pages/dca/positions/components/positions-list/current-positions/index.tsx

```diff
@@ -197,12 +187,7 @@ const CurrentPositions = ({ isLoading }: CurrentPositionsProps) => {
     }
   };
 
-  const positionsInProgress = currentPositions
-    .filter(({ toWithdraw, remainingSwaps }) => toWithdraw > 0n || remainingSwaps > 0n)
-    .sort(comparePositions);
-  const positionsFinished = currentPositions
-    .filter(({ toWithdraw, remainingSwaps }) => toWithdraw <= 0n && remainingSwaps <= 0n)
-    .sort(comparePositions);
+  const sortedPositions = orderBy(currentPositions, ['remainingSwaps', 'toWithdraw'], ['desc', 'desc']);
```

Comment:
Have you checked that this orders correctly? Had some issue with orderBy and bigints where it converted them to strings before, so it was doing string comparison